### ISSUE TITLE
Testing Micro Warp Jump

### ISSUE THREAD

**Issue by treker-ed** [RavienGaming/Reforged-Eden-2#229]
### Reforged Eden 2 Build Number

16

### Game mode

Survival

### Tested in vanilla

Not tested in vanilla or not applicable

### Description of issue

SP Only - MP not applicable.
Trying out the new micro warp jump. Feature is turned on as per the requirements via the admin panel.
1 x refined pentaxid is in personal inventory.
Sat in Epsilon IV sector, 11.54 km from target that I want to jump to. 
Initiate warp jump from pda - counts down, and then ... nothing! Message states that I need to have 1x refined pentaxid in personal inventory, however this was definitely in my inventory before initiating the jump. Seems to be using the pentaxid, but not jumping!

### Error code from console or log (Optional)

No error codes in console - but console is stating that the micro jump has completed - but ship i am in is still in its position sitting at 11.54 kms from its target.

### POI Name

N/A

### Playfield Name

Epsilon IV

### Playfield Class

OrbitEasy

### Star Name

Epsilon

### Star Class

MStartingSystem

### Mission Name

n/a

### Mission Objective

n/a

### Game Seed

987053

**Comment by RavienGaming** [comment]
Did you restart the game since you entered that playfield as per the instructions?

**Comment by treker-ed** [comment]
Not that particular playfield - however, I had done a PDA Readdata on restarting the game this morning and restarted the game then. This was the first time I had entered this particular playfield. I had not been in the epsilon 4 playfield before attempting to use the microjump. I entered the system and was placed 11.5 kms from a target and decided to try the microjump.

**Comment by RavienGaming** [comment]
Yes you have to restart the game since entering any playfield while piloting the ship due to a game bug with the API used.

**Comment by treker-ed** [comment]
OK... As I had already done that after the PDA readdata I had not realised I needed to do it again. 
As useful as this feature maybe, and of course, something many people have been asking for, that maybe a bit of a pain in the posterior if it needs to be done on entering every playfield that you would want to use the microjump. Thank you though for the work being put into this feature.

**Comment by RavienGaming** [comment]
Thank you for trying it out! The devs are aware of the API bug so hopefully they can fix it. I'll leave this open so others can more easily find it.

**Comment by RavienGaming** [comment]
The API bug has now been fixed in Empyrion, so you should no longer have to restart the game after changing playfields before using the micro jump drive.
